### PR TITLE
Upload correct order task

### PR DESCRIPTION
**Pull request by jagjeetst**
## Issue

When files are uploaded to a folder they should be inserted in the correct order instead of at the end #240

## Description

Fixed sorting of files after uploading them in upload area.
[video](https://www.loom.com/share/b3f0991f0765462297322d96d8aa5797)

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -550,11 +550,22 @@ function uploadFiles(files) {
...
       addFile(file);
     });
-
+    sortImageFiles();
     $progress.addClass('hidden');
   });
 }
 
+function sortImageFiles() {
+  var sort_by_name = function (a, b) {
```
Comment (squallstar) [inline js/interface.js]: please use camelCase for function names as per code standards

File: js/interface.js [hunk 2]
```diff
@@ -550,11 +550,22 @@ function uploadFiles(files) {
     files.forEach(function(file) {
       addFile(file);
     });
-
+    sortImageFiles();
     $progress.addClass('hidden');
   });
 }
 
+function sortImageFiles() {
```
Comment (squallstar) [inline js/interface.js]: why does it say "Image" in the name? Any type of file and folder should be sorted.

Note: the order to use should be Folders A-Z and then Files A-Z

e.g.

```
A (folder)
B (folder)
Z (folder)
A (file)
C (file)
c (file)
D (file)

File: js/interface.js (hunk 1 above)
Comment (jagjeetst) [reply js/interface.js]: Fixed.

File: js/interface.js (hunk 2 above)
Comment (jagjeetst) [reply js/interface.js]: Fixed.

File: js/interface.js [hunk 3]
```diff
@@ -522,7 +522,7 @@ function hideDropZone() {
   $dropZone.removeClass('active');
 }
 
-function uploadFiles(files) {
+function uploadFiles(files) {
```
Comment (squallstar) [inline js/interface.js]: please set your editor to trim empty spaces, on Sublime it's:

```
"translate_tabs_to_spaces": true,
"trim_trailing_white_space_on_save": true,
```

Comment (jagjeetst) [reply js/interface.js]: This is fixed now.

**Pull request closed without merging**